src/components/Sum.jsx

```diff
@@ -1 +1,12 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({ a, b }) {
+  return (
+    <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
+  );
+}
```

Comment:
- better to make functional components an arrow functions
- you should return right away if you don't have any calculations inside your arrow function
- better to move text to a separate line, it improves readability
```suggestion
export const Sum = ({ a, b }) => (
    <p>
      {`Sum of ${a} and ${b} is ${a + b}`}
    </p>
);
```